### ISSUE TITLE
UnsupportedOperationException in LiveChatUtil.java line 2567

### ISSUE THREAD

**Issue by alokvnair** [zoho/salesiq-mobilisten-android-sample#10]
After updating to 2.3, getting this below crash. Please look into this issue. Thanks!

```
Fatal Exception: android.view.InflateException: Binary XML file line #135: Binary XML file line #135: Error inflating class android.widget.EditText
       at android.view.LayoutInflater.inflate(LayoutInflater.java:551)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:429)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:380)
       at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:479)
       at android.app.Dialog.setContentView(Dialog.java:851)
       at com.zoho.livechat.android.utils.LiveChatUtil.showFeedbackDialog(LiveChatUtil.java:2567)
       at com.zoho.livechat.android.utils.LiveChatUtil$4.run(LiveChatUtil.java:2631)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:7325)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)

Caused by android.view.InflateException: Binary XML file line #135: Error inflating class android.widget.EditText
       at android.view.LayoutInflater.createView(LayoutInflater.java:657)
       at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
       at android.view.LayoutInflater.onCreateView(LayoutInflater.java:706)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:774)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:847)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:527)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:429)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:380)
       at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:479)
       at android.app.Dialog.setContentView(Dialog.java:851)
       at com.zoho.livechat.android.utils.LiveChatUtil.showFeedbackDialog(LiveChatUtil.java:2567)
       at com.zoho.livechat.android.utils.LiveChatUtil$4.run(LiveChatUtil.java:2631)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:7325)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)

Caused by java.lang.reflect.InvocationTargetException
       at java.lang.reflect.Constructor.newInstance(Constructor.java)
       at android.view.LayoutInflater.createView(LayoutInflater.java:631)
       at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
       at android.view.LayoutInflater.onCreateView(LayoutInflater.java:706)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:774)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:847)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:527)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:429)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:380)
       at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:479)
       at android.app.Dialog.setContentView(Dialog.java:851)
       at com.zoho.livechat.android.utils.LiveChatUtil.showFeedbackDialog(LiveChatUtil.java:2567)
       at com.zoho.livechat.android.utils.LiveChatUtil$4.run(LiveChatUtil.java:2631)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:7325)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)

Caused by java.lang.UnsupportedOperationException: Failed to resolve attribute at index 7: TypedValue{t=0x2/d=0x7f04038f a=3}
       at android.content.res.TypedArray.getColorStateList(TypedArray.java:484)
       at android.widget.TextView.<init>(TextView.java:1352)
       at android.widget.EditText.<init>(EditText.java:91)
       at android.widget.EditText.<init>(EditText.java:87)
       at android.widget.EditText.<init>(EditText.java:83)
       at java.lang.reflect.Constructor.newInstance(Constructor.java)
       at android.view.LayoutInflater.createView(LayoutInflater.java:631)
       at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
       at android.view.LayoutInflater.onCreateView(LayoutInflater.java:706)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:774)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:716)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:847)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:855)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:810)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:527)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:429)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:380)
       at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:479)
       at android.app.Dialog.setContentView(Dialog.java:851)
       at com.zoho.livechat.android.utils.LiveChatUtil.showFeedbackDialog(LiveChatUtil.java:2567)
       at com.zoho.livechat.android.utils.LiveChatUtil$4.run(LiveChatUtil.java:2631)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:7325)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)

```

**Comment by alokvnair** [comment]
@Angu-G We are getting too many crashes for this and https://github.com/zoho/salesiq-mobilisten-android-sample/issues/14 since moving to 2.3 and is taking down our crash-free users numbers. It would be great if you could fix these at the earliest and release a fixed version so that we can move to it.

**Comment by Angu-G** [comment]
@alokvnair We are working on a fix. It would be great if you confirm for us whether this issue and #14 occur only in API 19 (Kitkat) devices?

**Comment by alokvnair** [comment]
@Angu-G https://github.com/zoho/salesiq-mobilisten-android-sample/issues/14 happens only in API 19 but this one is not specific to one OS and is happening on multiple. Please find the distribution of the OS for this issue below.

<img width="428" alt="Screenshot 2020-10-06 at 11 00 54 AM" src="https://user-images.githubusercontent.com/8415897/95162471-6b96e380-07c3-11eb-91b5-6bd492dd24bd.png">

**Comment by Angu-G** [comment]
Thank you @alokvnair . Let me check this in our end and get back.

**Comment by Angu-G** [comment]
@alokvnair Thank you for your patience. We have submitted for testing with all the issues filed, can be expected to live in a day or two.

**Comment by alokvnair** [comment]
Thanks @Angu-G . Please do update here once the release is available.

**Comment by Angu-G** [comment]
@alokvnair Reported Issues have been fixed in version 2.3.1.

**dependency:**
implementation 'com.zoho.salesiq:mobilisten:2.3.1'

**Comment by alokvnair** [comment]
Thanks @Angu-G for the quick resolution. Will update to the new version in our release this week.
